changelog/front-commerce-3.0.mdx

```diff
@@ -0,0 +1,249 @@
+user experience. With GraphQL on the server, this is no longer a blocker.
+
+Front-Commerce now embraces modern web standards for forms, improving code
+accessibility and sustainability. We've replaced `formsy-react` with Remix
+`<Form>` component, aligned with standard form-related objects, and simplified
+the developer experience.
+
+We'll gradually ensure that every user interaction can _work without Javascript_
+as we refactor the theme for Progressive Enhancement.
+
+### CSS vs. Sass
+
+This release marks a first shift towards using CSS features over Sass, reducing
+our dependency on Sass. As the CSS standard continues to evolve, we're prepared
+to explore new theming possibilities in the coming years.
+
+We've decided to go step-by-step, and in this new major Front-Commerce version
+we've replaced the use of Sass variables with CSS variables.
+
+### Progressive Enhancement (PE)
+
+Progressive Enhancement will be at the heart of all Front-Commerce 3.x version
+onwards. We aim to deliver fast and enjoyable storefronts for all users, no
+matter their circumstances. Leveraging Remix features like routes, loaders, and
+actions, Front-Commerce is primed for progressive enhancement.
+
+In the future, storefronts could even be turned into **Multiple Page
+Applications** without JavaScript… who knows!
+_[(View Transitions API is an awesome web standard and people are building awesome things with it today)](https://astro-records.pages.dev/)_
+
+### A standard technical stack for futureproof projects
+
+Front-Commerce has always adhered to modern tech standards, adopting them
+because they were futureproof … not because they were trendy. We've adopted
+React, Node, GraphQL, and the Web Platform and have always tried to build on top
+of them with the best practices in mind.
+
+**Developing with Front-Commerce should feel like working with these
+technologies outside of Front-Commerce**. That's how agencies can recruit new
+developers to join their team and be quickly efficient on Front-Commerce
+projects. That's how we can guarantee that projects developed with
+Front-Commerce will be futureproof and could evolve, no matter where our
+industry is headed in the future.
+
+This year, AI and LLMs changed a lot of things in the way everyone works and
+interacts online. The relationship between standard technologies and AI-assisted
+development workflows might not be very clear at a first glance, but we're
+excited to incorporate this into your experience around Front-Commerce … but
+that's another story.
+
+## Welcome to v3
+
+If you're still reading, you must have plenty of questions about the next steps
+for you.
+
+We can't address anything in one blog post, but we're happy to help you out!
+
+Some highlights:
+
+- in the next weeks, we'll keep publishing concrete content to zoom on new
+  Front-Commerce concepts so you'll keep learning more about this. Follow us!
+- yes, we recommend that new Front-Commerce projects start with 3.0
```

Comment:
I would remove the `yes` here as no question has been asked, unless reformatting this section into a Q&A or FAQ

```suggestion
- We recommend that new Front-Commerce projects start with 3.0
```